explorer/src/providers/accounts/rewards.tsx

```diff
@@ -111,13 +115,20 @@ async function fetchRewards(
   const getInflationReward = async (epoch: number) => {
     try {
       const result = await connection.getInflationReward([pubkey], epoch);
-      return result[0];
+      if (result[0] !== null) {
+        return result[0];
+      }
     } catch (error) {
-      if (cluster !== Cluster.Custom) {
+      if (
+        cluster !== Cluster.Custom &&
+        !error.message.match(/Block not available/)
```

Comment:
Is this an actual response from this rpc method?

Reply:
The full response is something like 'Failed to fetch inflation reward: Block not available for...', which should no longer happen on my latest commit.